docs/Operators.md

```diff
@@ -15277,7 +15277,7 @@ x_original = length_resized > 1 ? start_x * (length_original - 1) + x_resized *
 <dt><tt>roi</tt> : T2</dt>
 <dd>1-D tensor given as [start1, ..., startN, end1, ..., endN], where N is the rank of X. The RoIs' coordinates are normalized in the coordinate system of the input image. It only takes effect when coordinate_transformation_mode is "tf_crop_and_resize"</dd>
 <dt><tt>scales</tt> : tensor(float)</dt>
-<dd>The scale array along each dimension. It takes value greater than 0. If it's less than 1, it's sampling down, otherwise, it's upsampling. The number of elements of 'scales' should be the same as the rank of input 'X'. Only one of 'scales' and 'sizes' can be specified. If 'size' is needed, the user can use an empty string as the name of 'scales' in this operator's input list.</dd>
+<dd>The scale array along each dimension. It takes value greater than 0. If it's less than 1, it's sampling down, otherwise, it's upsampling. The number of elements of 'scales' should be the same as the rank of input 'X'. Only one of 'scales' and 'sizes' can be specified. If 'size' is needed, the user can use a nonempty string as the name of 'scales' but set it to empty data (zero shape) in this operator's input list.</dd>
```

Comment:
Can this -> "If 'size' is needed, the user can use a nonempty string as the name of 'scales' but set it to empty data (zero shape) in this operator's input list."
be changed to ->
"If 'size' is specified, then set scales to empty data (zero shape) in this operator's input list."

Basically scales is not an optional input, adding empty data will imply that input name will be non-empty.